src/components/Resume/ResumeDetail/ResumeApi.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com/users/",
+});
+
+export async function GetUser(user) {
+  try {
+    let result = await (await instance.get(user)).data;
+    return result;
+  } catch (error) {
+    console.log(error);
```

Comment:
remove this console.log and any other that you find in your code please.
Those are only used for developers, we don't want to provide that information to final users.
Handle the error, you have to decide what you are going to do or return if you get an error while fetching data.